src/main/resources/application-development.properties

```diff
@@ -1,6 +1,6 @@
 logging.level.sql=DEBUG
 logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
-spring.datasource.url=jdbc:h2:file:./target/db-development
+spring.datasource.url=jdbc:h2:file:./target/db-development;AUTO_SERVER=TRUE
```

Comment:
This is the only change that should be in this PR.

You can make that happen by making a new branch off of`main` and then doing:

```
git cherry-pick 737005a5319341c92825c2daea590ba614e2e4af
```